Update te e infusi

marcelwandji opened the pull request.

marcelwandji committed:
aggiunta funzione getTeInfusi() al DBAccess

marcelwandji committed:
modifica te_infusi

dkokollari reviewed (changes requested):
Hai provato a validare i file con XHTML 1.0 Strict?

marcelwandji commented:
Non ho validato le pagine

dkokollari committed:
Rinominato getTeInfusi() -> getTeInfusiv1()
Conflitto certo con l'omonimo metodo offerto dal branch develop

dkokollari committed:
Corretto !DOCTYPE ad XHTML 1.0 Strict

dkokollari committed:
Modificato titolo in <head>

dkokollari committed:
Aggiunto meta keywords

dkokollari committed:
Aggiornato riferimento a favicon.ico in <head>

dkokollari committed:
Modificati riferimenti link ad elementi di stile (fonts google)

dkokollari committed:
Aggiunto type agli script

dkokollari committed:
Modificato <h1>

dkokollari committed:
Formattato il codice

dkokollari committed:
Corretto titolo in <head>

dkokollari committed:
Corretto <h1>

dkokollari committed:
Chiuso il tag bottone "Aggiungi"

dkokollari committed:
Abbellito il codice

dkokollari committed:
Indentato il codice

dkokollari committed:
Modificato titolo in <head>

dkokollari committed:
Aggiunto meta keywords

dkokollari committed:
Aggiornato riferimento a favicon.ico in <head>

dkokollari committed:
Modificati riferimenti link ad elementi di stile (fonts google)

dkokollari committed:
Aggiunto type agli script

dkokollari committed:
Aggiornato sezione header

dkokollari committed:
Aggiornato sezione content

dkokollari committed:
Rimossi script

dkokollari committed:
Rimossi alcuni spazi di indentazione

dkokollari committed:
Aggiunto div "container"
Bisogna includere il container per evitare errori di validazione sui 
bottoni "menu_btn" e "right-of-menu"

dkokollari committed:
Aggiunto avviso al codice che non valida

dkokollari committed:
Sostituito codice che non valida. Aggiunto placeholder

dkokollari committed:
Aggiunto header

dkokollari committed:
Aggiornata implementazione visualizzazione te e infusi

dkokollari committed:
Aggiunto riferimento a design.js

dkokollari committed:
Aggiunto ipertesto "Modifica" alla lista te e infusi

dkokollari committed:
Indentato il codice dinamico

dkokollari committed:
Abbellito il codice

dkokollari committed:
Corretta la visualizzazione dei caratteri speciali

dkokollari committed:
Aggiunte entità html nelle informazioni di te e infusi

dkokollari committed:
Merge branch 'develop' into update_te_e_infusi
